docker-compose.yml

```diff
@@ -1,103 +1,99 @@
-            - sail
-        depends_on:
-            - mysql
-            - redis
-            - meilisearch
-            - mailpit
-    mysql:
-        image: 'mysql/mysql-server:8.0'
+            - learnhub
+    nginx:
+        image: nginx:latest
+        container_name: LearnHub_nginx
+        restart: unless-stopped
         ports:
-            - '${FORWARD_DB_PORT:-3306}:3306'
-        environment:
-            MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
-            MYSQL_ROOT_HOST: '%'
-            MYSQL_DATABASE: '${DB_DATABASE}'
-            MYSQL_USER: '${DB_USERNAME}'
-            MYSQL_PASSWORD: '${DB_PASSWORD}'
-            MYSQL_ALLOW_EMPTY_PASSWORD: 1
+            - "8000:80"
         volumes:
-            - 'sail-mysql:/var/lib/mysql'
-            - './vendor/laravel/sail/database/mysql/create-testing-database.sh:/docker-entrypoint-initdb.d/10-create-testing-database.sh'
+            - ./:/var/www
+            - ./docker/nginx:/etc/nginx/conf.d
         networks:
-            - sail
-        healthcheck:
-            test:
-                - CMD
-                - mysqladmin
-                - ping
-                - '-p${DB_PASSWORD}'
-            retries: 3
-            timeout: 5s
+            - learnhub
     redis:
-        image: 'redis:alpine'
+        image: redis:latest
+        container_name: LearnHub_redis
+        restart: unless-stopped
         ports:
-            - '${FORWARD_REDIS_PORT:-6379}:6379'
+            - "6379:6379"
         volumes:
-            - 'sail-redis:/data'
+            - ./docker/redis:/data
         networks:
-            - sail
-        healthcheck:
-            test:
-                - CMD
-                - redis-cli
-                - ping
-            retries: 3
-            timeout: 5s
+            - learnhub
     meilisearch:
-        image: 'getmeili/meilisearch:latest'
+        image: getmeili/meilisearch:latest
+        container_name: LearnHub_meilisearch
         ports:
-            - '${FORWARD_MEILISEARCH_PORT:-7700}:7700'
+            - "7700:7700"
         environment:
-            MEILI_NO_ANALYTICS: '${MEILISEARCH_NO_ANALYTICS:-false}'
+            - MEILI_MASTER_KEY=your_master_key_here
         volumes:
-            - 'sail-meilisearch:/meili_data'
+            - ./docker/meilisearch/data:/data.ms
+        networks:
+            - learnhub
+    phpmyadmin:
+        image: phpmyadmin/phpmyadmin
+        container_name: LearnHub_phpmyadmin
+        environment:
+            - PMA_HOST=db
+            - PMA_PORT=3306
+        ports:
+            - "8080:80"
+        depends_on:
+            - db
         networks:
-            - sail
-        healthcheck:
-            test:
-                - CMD
-                - wget
-                - '--no-verbose'
-                - '--spider'
-                - 'http://localhost:7700/health'
-            retries: 3
-            timeout: 5s
-    mailpit:
-        image: 'axllent/mailpit:latest'
+            - learnhub
+    mailhog:
```

Comment:
Mailpit is the new and better version of Mailhog so we can use that one instead.